ontology/tool/tool.ttl

```diff
@@ -325,15 +325,15 @@ tool:Tool
 	rdfs:comment "A tool is an element of hardware and/or software utilized to carry out a particular function."@en ;
 	sh:property
 		[
-			sh:datatype xsd:anyURI ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:references ;
+			sh:class core:IdentityAbstraction ;
```

Comment:
core:IdentityAbstraction should only be used within the core namespace. It is a mechanism to allow references to Identity within core without causing cyclical importing.
In the tool namespace, uses such as this should directly use identity:Identity.

Reply:
Resolved in d7a593e.